src/components/application_manager/src/app_service_manager.cc

```diff
@@ -386,6 +396,24 @@ std::pair<std::string, AppService> AppServiceManager::FindServiceByName(
   return std::make_pair(std::string(), empty);
 }
 
+std::pair<std::string, AppService> AppServiceManager::FindServiceByAppID(
+    std::string name, std::string type) {
```

Comment:
Change `name` to `app_id` or `policy_app_id`. I thought it was referring to the service name at first.